migration-engine/core/tests/migration_tests.rs

```diff
@@ -230,7 +230,12 @@ fn changing_the_type_of_an_id_field_must_work() {
         assert_eq!(column.tpe.family, ColumnTypeFamily::Int);
         assert_eq!(
             table.foreign_keys,
-            vec![ForeignKey {
+            &[ForeignKey {
+                constraint_name: match sql_family {
+                    SqlFamily::Postgres => Some("A_b_fkey".to_owned()),
+                    SqlFamily::Mysql => Some("A_ibfk_1".to_owned()),
+                    SqlFamily::Sqlite => None,
+                },
```

Comment:
This is what I was complaining about yesterday - we have to match on the SqlFamily and put in the default database constraint names here. But it's not as noisy as I had feared, so it might be ok.